curriculum/challenges/english/01-responsive-web-design/css-grid-magazine/part-056.md

```diff
@@ -0,0 +1,325 @@
+---
+id: 6148ceaf5d897d4d8b3554b3
+title: Part 56
+challengeType: 0
+dashedName: part-56
+---
+
+# --description--
+
+Give the `.text` selector a `text-align` property set to `justify.`
```

Comment:
We may want to consider removing this step. According to [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/text-align#accessibility_concerns), "The inconsistent spacing between words created by justified text can be problematic for people with cognitive concerns such as Dyslexia."

Reply:
Maybe worth adding a note about the accessibility?

The justify does make the text look good.